{{cookiecutter.app_name}}/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "{{cookiecutter.app_name}}",
+  "version": "1.0.0",
+  "description": "{{cookiecutter.project_short_description}}",
+  "main": "js/main.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "NODE_ENV=prod ./node_modules/.bin/webpack --progress --colors -p",
+    "start": "./node_modules/.bin/webpack-dev-server --port 2992 --hot --inline",
```

Comment:
It might be nice to have this run both the webpack dev server and the flask debug server. [concurrently](https://github.com/kimmobrunfeldt/concurrently) is useful for this.